nucleus/security/core/pom.xml

```
@@ -228,4 +228,41 @@
             <artifactId>javassist</artifactId>
         </dependency>
     </dependencies>
+
+    <profiles>
+        <profile>
+            <id>UpdateTruststore</id>
```

It's 'trust store' rather than 'truststore' (that's just programmers being lazy), so an extra capital letter
```suggestion
            <id>UpdateTrustStore</id>
```